Lib/csv.py

```diff
@@ -394,6 +394,8 @@ def has_header(self, sample):
         # can't be determined, it is assumed to be a string in which case
         # the length of the string is the determining factor: if all of the
         # rows except for the first are the same length, it's a header.
+        # when the strings have varying length, the average length of all
```

Comment:
```suggestion
        # When the strings have varying length, the average length of all
```

Minor nitpick, since this comment uses complete sentences and sentence case throughout.